app/templates/components/scripts.twig

```diff
@@ -0,0 +1,6 @@
+{% block javascript %}
+    <script src="<?= $_ENV['BASE_URL']?>/web-assets/js/jquery-3.5.1.min.js"></script>
```

Comment:
All of our scripts are here! 
If you needed to add a script for a single page, you could use the following:
```
{% block javascript %]
{{ parent() }}
<script src="web-assets/js/special_script.js"></script>
{% endblock %}
```

This will still load all of our scripts here, but add the "special_script" as well!